scripts/presets/travis/replica_sets/replicaset.json

```diff
@@ -0,0 +1,66 @@
+{
+    "id": "REPLICASET",
+    "name": "mongod",
+    "members": [
+        {
+            "procParams": {
+                "dbpath": "/tmp/REPLICASET/3000/",
+                "ipv6": true,
+                "logappend": true,
+                "logpath": "/tmp/REPLICASET/3000/mongod.log",
+                "journal": true,
```

Comment:
I expect this was copied over from our non-Travis configs. It looks like [`storage.journal.enabled`](https://docs.mongodb.com/manual/reference/configuration-options/#storage.journal.enabled) defaults to true only on 64-bit platforms, so it's probably sensible to keep this set explicitly in case.